### PR TITLE
[#7] Better help command output

### PR DESCRIPTION
**Pull request by williankeller**
Adding more detail about each command and how to use it as requested under https://github.com/php-school/workshop-manager/issues/7 issue.
Feel free to suggest new comments or a new order for items to be displayed

### CODE REVIEW

File: app/config.php [hunk 1]
```diff
@@ -37,25 +37,79 @@
     Application::class => \DI\factory(function (ContainerInterface $c) {
         $application = new \PhpSchool\WorkshopManager\Application('PHP School workshop manager', '1.1.0', $c);
         $application->command('install workshopName', InstallWorkshop::class, ['add'])
-            ->setDescription('Install a PHP School workshop.');
+            ->setDescription('Install a PHP School workshop.')
+            ->setHelp(<<<'EOF'
+This command requires a <comment>workshopName</comment> code as argument:
```
Comment (AydinHassan) [inline app/config.php]: as `an` argument

File: app/config.php [hunk 2]
```diff
@@ -37,25 +37,79 @@
...
+  <info>workshop-manager <comment>workshopName</comment></info>
```
Comment (AydinHassan) [inline app/config.php]: I think this is missing `%command.name%`

File: app/config.php (hunk 1 above)
Comment (AydinHassan) [reply app/config.php]: Same as all the other commands also

**Review by AydinHassan — CHANGES_REQUESTED** [review]
Thanks @williankeller this is really great! Just a few small requests please :)

**Comment by AydinHassan** [comment]
Ignore the travis failure - this is due to the auto phar building and you not having access to the secrets. There's probably a way to disable that on PR's but I don't know how yet :D

File: app/config.php (hunk 2 above)
Comment (williankeller) [reply app/config.php]: Yes, it was, my bad.

**Comment by williankeller** [comment]
New commit was made [#7] Fix typo & missing placeholder 1d10b9b

**Review by AydinHassan — APPROVED** [review]

**Comment by AydinHassan** [comment]
Thank you @williankeller ! `self-update` in a few minutes and your changes should come through automatically!

**Pull request merged**